app/grandchallenge/reader_studies/forms.py

```diff
@@ -96,6 +99,8 @@ class Meta:
         help_texts = READER_STUDY_HELP_TEXTS
         widgets = {
             "description": TextInput,
+            "modalities": Select2MultipleWidget,
+            "structures": Select2MultipleWidget,
         }
```

Comment:
Did you leave out the `publications` field here on purpose?

Reply:
No, nice find!

Reply:
We really should use the same form for create and update...